sdk/servicebus/azure-servicebus/tests/test_sessions.py

```diff
@@ -731,6 +731,27 @@ def lock_lost_callback(renewable, error):
             renewer.close()
             assert len(messages) == 2
 
+        # test voluntary halt of auto lock renewer when session is closed
```

Comment:
hey @rakshith91, does this test look similar to what you did when you were [trying to reproduce](https://github.com/Azure/azure-sdk-for-python/issues/14336#issuecomment-721246436)?

Reply:
yes - this seems right to me

Reply:
looks good to me -- we should cover async well.